app/ui-react/packages/api/package.json

```diff
@@ -60,8 +59,8 @@
     "immer": "^8.0.1",
     "monaco-editor-core": "^0.19.3",
```

Comment:
I didn't realize that we still had monaco references in here, these should be leftovers from removing the datavirt components.  I wonder if cleaning up these along with any relevant craco config (the `"vscode": require.resolve('monaco-languageclient/lib/vscode-compatibility'),` line) could help at all with the build error?  I can look at removing these in a separate PR tomorrow.

Reply:
Good to know, I'll see if I can remove them while I'm addressing the rest of the cleanup.